Rubberduck.Parsing/VBA/IBuiltInDeclarationLoader.cs

```diff
@@ -0,0 +1,10 @@
+﻿
+namespace Rubberduck.Parsing.VBA
+{
+    public interface IBuiltInDeclarationLoader
```

Comment:
We've replaced `Declaration.IsBuiltIn` with `Declaration.IsUserDefined`, and inverted the callers' logic accordingly, because "built-in" didn't feel right in light of COM references (vs formerly hard-coded "built-in" declarations); that would be `ICOMDeclarationLoader`... but then `ICOMReferenceManager` seems to share the same concern... feels like one of the two should end up discarded.